src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjectById.cs

```diff
@@ -28,15 +29,18 @@ public GetProjectById(IProjectsService projectsService)
         public IActionResult Run(
         [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "v1/projects/{id:guid}")] HttpRequest req, Guid id)
         {
-            var result = this.projectsService.GetProject(id);
-
-            if (result == null)
+            try
             {
-                return new NotFoundObjectResult(result);
+                var result = this.projectsService.GetProject(id);
+                return new OkObjectResult(result);
             }
-            else
+            catch (ProjectsException e)
             {
-                return new OkObjectResult(result);
+                return new ContentResult
+                {
+                    StatusCode = e.StatusCode,
+                    Content = e.ProjectsErrorMessage,
```

Comment:
You need to return an object with this format:
{
  "message" : "message of error"
}